skrifa/src/util.rs

```diff
@@ -0,0 +1,151 @@
+//! Internal helpers.
+
+use core::fmt;
+
+/// SmallVec like implementation but for fixed sized arrays whose
+/// size is only known at runtime.
+///
+/// Used to avoid allocations when possible for small or temporary
+/// arrays.
+///
+/// Consider taking on a smallvec dependency in the future.
```

Comment:
FWIW, just checked at least it's not approved and included as a third_party dependency in Chromium yet. Would be easier if it was. https://source.chromium.org/chromium/chromium/src/+/main:third_party/rust/?q=third_party%2Frust%2F%20